packages/protocol/contracts/stability/Reserve.sol

```diff
@@ -302,10 +307,50 @@ contract Reserve is
    * @param spender The address that is to be no longer allowed to spend Reserve funds.
    */
   function removeSpender(address spender) external onlyOwner {
-    isSpender[spender] = false;
+    delete isSpender[spender];
     emit SpenderRemoved(spender);
   }
 
+  function isAllowedToSpendExchange(address spender) public view returns (bool) {
+    return
```

Comment:
Why is this function necessary? Please include a comment explaining why `isExchangeSpender` is not enough, and the conditions that need to be met for this function to be deprecated